File: app/src/main/java/com/metahumanz/pacilread/stats/annual/AnnualReportTheme.java
package com.metahumanz.pacilread.stats.annual;

public enum AnnualReportTheme {
    LIGHT("light", "浅色"),
    DARK("dark", "深色");

    public final String slug;
    public final String label;

    AnnualReportTheme(String slug, String label) {
        this.slug = slug;
        this.label = label;
    }
}

File: app/src/main/kotlin/com/metahumanz/pacilread/stats/annual/AnnualReportTheme.kt
package com.metahumanz.pacilread.stats.annual

enum class AnnualReportTheme(
    @JvmField val slug: String,
    @JvmField val label: String,
) {
    LIGHT("light", "浅色"),
    DARK("dark", "深色"),
}
